components/template/Templates.vue

```diff
@@ -118,15 +119,12 @@ export default {
   .template_container {
     flex-direction: column;
   }
-  .left-sidebar {
-    min-height: auto;
-    box-shadow: 0px 4px 8px 0px rgba(0, 0, 0, 0.04);
-    position: sticky;
-    top: 54px;
-    padding: 1rem 0;
-  }
   .heading {
-    margin-top: 12px;
+    display: none;
```

Comment:
let this heading also be here for smaller screen sizes as well. No need to make it display: none;